README.md

```diff
@@ -70,8 +50,13 @@ $ terraform init
 $ terraform plan
 $ terraform apply
 ```
-## Note :
-ClientSecret can be obtained by running
-```sh
-$ terraform output ClientSecret
-```
+## Note:
+* The user should have `owner` role to create resources
+* Once terraform is applied successfully, it will create `client_credentials.json` file.
+
```

Comment:
Any additional notes to explain to customer that they have to provide these details back at Uptycs UI?

Reply:
Let me review that.